commercetools-models/src/main/java/io/sphere/sdk/products/search/PriceSelection.java

```diff
@@ -21,6 +21,10 @@
     String getPriceCustomerGroup();
     @Nullable
     String getPriceChannel();
+    @Nullable
+    String getStoreProjection();
+    @Nullable
+    String getLocaleProjection();
```

Comment:
The docs explicitly states that we support multiple `localeProjection` query parameters here (see ) and that means that the type should be `List<String>` here instead of just a `String`. This then requires further changes, which I think you fix on your own. I will just add comments below to tell you how the API should look like so that we follow the existing, but undocumented naming conventions of our JVM SDK.